watertap/unit_models/tests/test_clarifier.py

```diff
@@ -0,0 +1,187 @@
+        m.fs.unit.inlet.conc_mass_comp[0, "X_I"].fix(92 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_S"].fix(363 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_BH"].fix(50 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_BA"].fix(0 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_P"].fix(0 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "S_O"].fix(0 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "S_NO"].fix(0 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "S_NH"].fix(23 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "S_ND"].fix(5 * units.g / units.m**3)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_ND"].fix(16 * units.g / units.m**3)
+
+        # Alkalinity was givien in mg/L based on C
+        m.fs.unit.inlet.alkalinity[0].fix(7 * units.mol / units.m**3)
+
+        # Unit option
+        m.fs.unit.split_fraction[0, "effluent", "H2O"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "S_I"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "S_S"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "X_I"].fix(0.5192)
+        m.fs.unit.split_fraction[0, "effluent", "X_S"].fix(0.5192)
+        m.fs.unit.split_fraction[0, "effluent", "X_BH"].fix(0.5192)
+        m.fs.unit.split_fraction[0, "effluent", "X_BA"].fix(0.5192)
+        m.fs.unit.split_fraction[0, "effluent", "X_P"].fix(0.5192)
+        m.fs.unit.split_fraction[0, "effluent", "S_O"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "S_NO"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "S_NH"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "S_ND"].fix(0.993)
+        m.fs.unit.split_fraction[0, "effluent", "X_ND"].fix(0.5192)
+        m.fs.unit.split_fraction[0, "effluent", "S_ALK"].fix(0.993)
+
+        return m
+
+    @pytest.mark.unit
+    def test_dof(self, clarifier_frame):
+        m = clarifier_frame
+        assert degrees_of_freedom(m) == 0
+
+    @pytest.mark.unit
+    def test_units(self, clarifier_frame):
+        assert_units_consistent(clarifier_frame)
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_initialize(self, clarifier_frame):
+        initialization_tester(clarifier_frame)
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_solve(self, clarifier_frame):
+        m = clarifier_frame
+        results = solver.solve(m)
+
+        # Check for optimal solution
+        assert_optimal_termination(results)
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_circular_clarifier_costing(self, clarifier_frame):
```

Comment:
should we add a test without specifying the costing method to make sure the default costing method works?

Reply:
The default is set as circular, I test on my on workstation and it works find, and we can also directly delete the costint_method here